CMake fails with `LLVM_ENABLE_SPHINX=ON`

```
CMake Error at interpreter/cling/docs/CMakeLists.txt:60 (include):
  include could not find requested file:

    AddSphinxTarget
```

when configuring with

`cmake -DLLVM_ENABLE_SPHINX=ON`

Affects `master` and latest v6.28/06 / `v6-28-00-patches`.

Reported on the forum: https://root-forum.cern.ch/t/cmake-failed-w-dllvm-enable-sphinx-on/56260

--- a/interpreter/CMakeLists.txt
+++ b/interpreter/CMakeLists.txt
@@ -237,6 +237,7 @@ if(builtin_llvm)
     ${CMAKE_BINARY_DIR}/interpreter/llvm-project/llvm/include
     CACHE STRING "LLVM include directories."
     )
+  list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/llvm-project/llvm/cmake/modules")
   #---Set into parent scope LLVM_VERSION --------------------------------------------------------------
   get_directory_property(_llvm_version DIRECTORY llvm-project/llvm DEFINITION LLVM_VERSION)
   set(LLVM_VERSION "${_llvm_version}" PARENT_SCOPE)

